src/modules/app/StateType.js

```diff
@@ -68,38 +68,27 @@ export const defaultCitiesState: CitiesStateType = {
 }
 
 export type SearchRouteType = {|
-  +categoriesMap: CategoriesMapModel | null
+  +categoriesMap: CategoriesMapModel
 |}
 
 export type CityContentStateType = {|
-  +lastUpdate: Moment | null,
```

Comment:
Where/how do we deal with the lastUpdate now?

Reply:
Just like it was before since this is just the redux state. We didn't even put any values there or read from it, so I figured it can be removed.